bigday/templates/home.html

```diff
@@ -45,7 +45,9 @@
     <header id="main-header">
         <div class="header-content">
             <div class="header-content-inner">
-                <h1>9.1.19</h1>
+
+                <!-- Inbetween <h1> and </h1> put the date to be displayed in front of photo-->
+                <h1>  </h1>
```

Comment:
You can just delete these heading tags.

Reply:
Wanted to leave it in just in case we want to add it later if we change pictures.